app/models/branding_group.rb

```diff
@@ -76,4 +77,13 @@ def remove_branding_association
   def remove_cached_images
     UserAssetService.remove_assets_from_remote("branding_groups/#{self.id}")
   end
+
+  def self.visible_groups_to_user(user)
+    if user.present?
```

Comment:
I appreciate that the way you've constructed it is more Ruby-y.  I've started relying on 'truthiness' of objects less lately because I found I was writing a lot of bugs where I'd forget that e.g. an empty array is false-y but an empty hash is truth-y, or that 0 is truthy in Ruby but false-y in JS.  But that's just my senility approaching...

As for the ternary, I've started avoid ternaries except for trivial assignment is done, b/c it makes it much easier to put debug statements in and check which way the branch is going.  e.g. 
```
 foo = user.admin? ? bar : baz. 
```
works well for me, but for 
```
foo = user.admin? ? bar : Study.where(....)
```
I'd prefer the query on a separate line so it's easier to isolate.  But like you said, these are just preferences